CONTRIBUTING.md

```diff
@@ -0,0 +1,216 @@
+# Contributing to win32-certstore
+
+We're glad you want to contribute to win32-certstore! The first
```

Comment:
Use We are instead of We're. I am not sure about the standards but see other chef docs for this.